neuromation/cli/parse_utils.py

```diff
@@ -46,3 +53,100 @@ def parse_memory(memory: str) -> int:
 
 def to_megabytes(value: str) -> int:
     return int(parse_memory(value) / (1024 ** 2))
+
+
+@dataclasses.dataclass(frozen=True)
+class JobColumnInfo:
+    id: str
+    title: str
+    align: Align
+    width: ColumnWidth
+
+
+COLUMNS = [
+    JobColumnInfo("id", "ID", Align.LEFT, ColumnWidth()),
+    JobColumnInfo("name", "NAME", Align.LEFT, ColumnWidth(max=20)),
+    JobColumnInfo("status", "STATUS", Align.LEFT, ColumnWidth(max=10)),
+    JobColumnInfo("when", "WHEN", Align.LEFT, ColumnWidth(max=15)),
+    JobColumnInfo("image", "IMAGE", Align.LEFT, ColumnWidth(max=40)),
+    JobColumnInfo("owner", "OWNER", Align.LEFT, ColumnWidth(max=25)),
+    JobColumnInfo("cluster_name", "CLUSTER", Align.LEFT, ColumnWidth(max=15)),
+    JobColumnInfo("description", "DESCRIPTION", Align.LEFT, ColumnWidth(max=50)),
+    JobColumnInfo("command", "COMMAND", Align.LEFT, ColumnWidth(max=100)),
+]
+
+COLUMNS_MAP = {column.id: column for column in COLUMNS}
+
+COLUMNS_RE = re.compile(
+    r"""
+    (?P<col>(\{[^}]+)\})|
```

Comment:
Inner `(` and `)` are not needed.

Besides, it may be more convenient to use `\{(?P<col>[^}]+)\}|`, so you will not need to strip the first and the last characters by `[1:-1]`.

Reply:
done